Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -5153,6 +5154,25 @@ static VisiblePositionRange constrainRangeToSelection(const VisiblePositionRange
         scheduleEditorStateUpdateForStartOrEndContainerNodeIfNeeded(endContainer.get());
 }
 
+FloatSize WebPage::screenSizeForHeadlessMode(const LocalFrame&, FloatSize defaultSize) const
+{
+    if (!currentUserInterfaceIdiomIsSmallScreen())
+        return m_viewportConfiguration.minimumLayoutSize();
+
+    static constexpr std::array fixedSizes {
+        FloatSize { 320,  568 },
```

Comment:
Where did these values come from?

Reply:
These values represent device metrics from several iPhone models. While somewhat arbitrary, I did make sure to pick the most prevalent width values, by number of models with that width.